openapi_python_client/templates/client.py.jinja

```diff
@@ -41,6 +43,8 @@ class Client:
     _cookies: Dict[str, str] = field(factory=dict, kw_only=True)
     _headers: Dict[str, str] = field(factory=dict, kw_only=True)
     _timeout: Optional[httpx.Timeout] = field(default=None, kw_only=True)
+    _retries: Optional[int] = field(default=None, kw_only=True)
+    _transport: Optional[httpx.HTTPTransport] = field(default=None, kw_only=True)
```

Comment:
It looks like `retries` cannot be set on `HTTPTransport` after construction—so there'd be no way for us to take _both_ a transport _and_ a retries and respect both. I'm tempted to say we should take only a transport (like the underlying clients do).